Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -593,6 +594,20 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
     if (m_failureScheduled)
         return;
 
+    if (CheckedPtr session = m_session.get()) {
+        CheckedPtr storageSession = session->networkStorageSession();
+        if (storageSession && storageSession->cookiesVersion() < m_requiredCookiesVersion) {
+            RELEASE_LOG(Loading, "%p - NetworkDataTaskCocoa::resume is delayed because cookies version (%" PRIu64 ") is lower than required (%" PRIu64 ")", this, storageSession->cookiesVersion(), m_requiredCookiesVersion);
+            storageSession->addCookiesVersionChangeCallback(m_requiredCookiesVersion, [weakThis = ThreadSafeWeakPtr { *this }] {
+                if (auto protectedThis = weakThis.get()) {
+                    RELEASE_LOG(Loading, "%p - NetworkDataTaskCocoa::resume delayed task is started", protectedThis.get());
```

Comment:
This should mention cookies. from the logs, it will not be clear why the load is resuming now.